app/src/main/res/values/color_defs.xml

```diff
@@ -23,8 +23,10 @@
   <color name="color_def_white_light">#F9F9F9</color>
   <color name="color_def_accessible_grey">#333333</color>
   <color name="color_def_accessible_light_grey">#555555</color>
+  <color name="color_def_accessible_light_grey_100">#666666</color>
```

Comment:
Rename to `color_def_accessible_light_grey_2`

Reply:
This appears to have been resolved.